### PR TITLE
DIE WHITESPACE DIE

### PR DESCRIPTION
**Pull request by coiax**
It's german for: "I'll only be able to get away with this during a
feature freeze."

Notes:
- DMM2TGM.py no longer inserts space on conversion

### CODE REVIEW

File: html/changelog.html [hunk 1]
```diff
@@ -5,16 +5,16 @@
   <link rel="stylesheet" type="text/css" href="changelog.css">
   <base target="_blank" />
   <script type='text/javascript'>
-
```
Comment (Iamgoofball) [inline html/changelog.html]: # DO NOT ADJUST THE GENERATED CHANGELOG IN ANY WAY

i repeat

# DO NOT ADJUST THE GENERATED CHANGELOG IN ANY WAY

File: html/create_object.html [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
-		
+
 		A <input type="radio" name="offset_type" value="absolute">
 		R <input type="radio" name="offset_type" value="relative" checked="checked"><br>
-		
+
 		Number: <input type="text" name="object_count" value="1" style="width:30px">
 		Dir: <input type="text" name="object_dir" value="2" style="width:30px">
 		Name: <input type="text" name="object_name" value="" style="width:180px"><br>
-		Where:
```
Comment (Iamgoofball) [inline html/create_object.html]: Be careful with this shit m8

File: tools/ss13_genchangelog.py [hunk 3]
```diff
@@ -70,7 +70,7 @@ def dictToTuples(inp):
         with open(changelog_cache) as f:
             (_, all_changelog_entries) = yaml.load_all(f)
             failed_cache_read = False
-
```
Comment (Iamgoofball) [inline tools/ss13_genchangelog.py]: # DO NOT TOUCH THE CHANGELOG GENERATOR

i repeat

# DO NOT TOUCH THE CHANGELOG GENERATOR

**Comment by Shadowlight213** [comment]
This needs it's own freeze and it's going to conflict with literally every pr.

**Comment by KorPhaeron** [comment]
Could we not do this again?

**Comment by lordpidey** [comment]
Wait what?  Why did this fail travis?

Travis's detailed logs say it compiled just fine, yet it gives the red X of failure.

**Comment by Iamgoofball** [comment]
@lordpidey Changelog files broke.

**Comment by PKPenguin321** [comment]
why tho

**Comment by MrStonedOne** [comment]
because python is a whitespace sensitive language.

Also, can we not do this..... Like please.

Its fucking annoying when you want to move down to the next line in the middle of an existing proc to add a line, and it's not indented.

**Comment by optimumtact** [comment]
Okay so first of all this only strips extra whitespace from the end of lines and does not change the end of line settings like I attempted and will DO ONE DAY AGAIN MARK MY WORDS. @Shadowlight213 @KorPhaeron @PKPenguin321 

This should be very unlikely to conflict to any PR and will not affect any tool or file in use as extraneous space on the ends of lines is not a syntax issue

@MrStonedOne your editor should insert spacing for you automatically, does DreamEditor not do it?

**Comment by optimumtact** [comment]
@coiax if you want to adjust the example changelog you need to md5sum the updated version and plug the value into the .travis.yml file

**Comment by MrStonedOne** [comment]
oranges, it inserts then on new lines, but not when moving around with the arrow keys in existing blank lines in the middle of the proc.

**Comment by Incoming5643** [comment]
It's not an enforceable code standard

File: tools/dmitool/git_merge_installer.bat [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @echo off
-set tab=
```
Comment (Jordie0608) [inline tools/dmitool/git_merge_installer.bat]: Idk what this tool does but this removal here is probably a bad idea.

**Comment by coiax** [comment]
1) We can put a check in the travis to check for any trailing whitespace.
2) We can actually put common travis failure checks in a pre-commit git hook to stop some of the more common ones (like step_[xy])
3) We have a dmimerger at `tools/dmitool/` ?

**Comment by optimumtact** [comment]
1) yes
2) good luck with that with our coders
3) yes

File: tools/dmitool/git_merge_installer.bat (hunk 4 above)
Comment (Man-Shroom-) [inline tools/dmitool/git_merge_installer.bat]: ahahahaha

**Comment by Man-Shroom-** [comment]
WHITESPACE DID NOTHING WRONG

**Comment by coiax** [comment]
I don't care this much with any sort of opposition

**Pull request closed without merging**